src/httpserver.cpp

```diff
@@ -776,8 +776,10 @@ bool HttpServer::matchUrl(const QStringList& routeParts, const QString& path, QU
       // Proceed to grab the regular expression inside the expressed route.
       // TODO: TEST THIS
```

Comment:
Yeah? I noticed that=) Just made a very little contribution, cause' I'm using your code=) Very good work!